google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionPoolStressTest.java

```diff
@@ -92,6 +93,7 @@ private void setupSpanner(DatabaseId db) {
     when(spannerOptions.getNumChannels()).thenReturn(4);
     when(spannerOptions.getDatabaseRole()).thenReturn("role");
     SessionClient sessionClient = mock(SessionClient.class);
+    when(sessionClient.getSpanner()).thenReturn(mockSpanner);
```

Comment:
Curious as to why we need this mocking with current changes and did not require it earlier.

Reply:
The `isUnbalanced` method that was added to the session pool calls this method. See https://github.com/googleapis/java-spanner/pull/2442/files#r1318112669